### PR TITLE
panel: Check OS architecture

### LINKED ISSUE #158: panel: check OS architecture
**Describe the bug**
The script can be run on other architectures then `x86_64`, even though `MariaDB` only supports `x86_64`.

**To Reproduce**
Steps to reproduce the behaviour:
1. Run the script on 32 bit machine
2. The script tries to install `MariaDB` and can't

**Expected behaviour**
It should check if the OS is `x86_64` and disallow other architectures.

**OS (please complete the following information):**
 - OS: all
 - Installation: panel (wings already checks)

**Additional context**
```[error] The MariaDB Repository only supports x86_64 and aarch64 (detected i686).
[error] The MariaDB Repository supports these Linux OSs, on x86-64 only:
    * RHEL/CentOS 7 & 8
    * Ubuntu 16.04 LTS (xenial), 18.04 LTS (bionic), & 20.04 LTS (focal)
    * Debian 9 (stretch) & 10 (buster)
    * SLES 12 & 15
[error] See https://mariadb.com/kb/en/mariadb/mariadb-package-repository-setup-and-usage/#platform-support```

### PR DESCRIPTION
**Pull request by Linux123123**
Add OS architecture checking. Exactly the same as in wings.

Fixes #158

### CODE REVIEW

**Review by sinmineryt — APPROVED** [review]

**Comment by sinmineryt** [comment]
Bugfixes are always good :^)

**Comment by vilhelmprytz** [comment]
Thank you for the contribution! Do we need to detect the CPU architecture for panel? In what way will the panel not run on anything else than x86_64?

**Comment by sinmineryt** [comment]
See #158

**Comment by Linux123123** [comment]
Look at the #158.

**Comment by vilhelmprytz** [comment]
I realize I missed the obvious reference to the issue, sorry about that 🙂

File: install-panel.sh [hunk 1]
```diff
@@ -275,6 +275,20 @@ detect_distro() {
 }
 
 check_os_comp() {
+  MACHINE_TYPE=$(uname -m)
```
Comment (vilhelmprytz) [inline install-panel.sh:278]:
```suggestion
  CPU_ARCHITECTURE=$(uname -m)
```

File: install-panel.sh [hunk 2]
```diff
@@ -275,6 +275,20 @@ detect_distro() {
...
+  if [ "${MACHINE_TYPE}" != "x86_64" ]; then # check the architecture
```
Comment (vilhelmprytz) [inline install-panel.sh:279]:
```suggestion
  if [ "${CPU_ARCHITECTURE}" != "x86_64" ]; then # check the architecture
```

File: install-panel.sh [hunk 3]
```diff
@@ -275,6 +275,20 @@ detect_distro() {
...
+    print_warning "Detected architecture $MACHINE_TYPE"
```
Comment (vilhelmprytz) [inline install-panel.sh:280]:
```suggestion
    print_warning "Detected CPU architecture $CPU_ARCHITECTURE"
```

**Review by vilhelmprytz — COMMENTED** [review]
I realize that my suggestions might not be consistent with the Wings script. If this is the case, feel free to submit a PR 🙂

**Review by vilhelmprytz — APPROVED** [review]
LGTM

**Pull request merged**